### PR TITLE
Dbt freshness debug clean

### PR DESCRIPTION
**Pull request by kiliangojek**
# TL;DR
During my own testing from my repository, I realized that the run_results.json and manifest.json aren't automatically generated when running dbt source freshness

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
I removed the attributes from the raw_run_result and raw_manifest from the DBTFreshnessOutput dataclass. I also removed these outputs from the tests.

## Tracking Issue
_NA_

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: plugins/flytekit-dbt/flytekitplugins/dbt/schema.py [hunk 1]
```diff
@@ -246,16 +246,4 @@ def to_args(self) -> List[str]:
 @dataclass_json
 @dataclass
 class DBTFreshnessOutput(BaseDBTOutput):
```
Comment (eapolinario) [inline plugins/flytekit-dbt/flytekitplugins/dbt/schema.py:248]: should we have the raw logs coming from `dbt source freshness` ?

Comment (kiliangojek) [reply plugins/flytekit-dbt/flytekitplugins/dbt/schema.py]: Done 😄

**Comment by eapolinario** [comment]
@kiliangojek , can you merge master?

**Comment by kiliangojek** [comment]
> @kiliangojek , can you merge master?

Done :)

**Review by eapolinario — APPROVED** [review]
